src/pages/Article/ArticlePage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PageLayout from '../../components/PageLayout';
+import Button from "@mui/material/Button";
+import { Box } from '@mui/system';
+
+/**
+ * Renders the Article page.
+ * @returns {JSX.Element} ArticlePage component
+ */
+function ArticlePage() {
+  return (
+	<PageLayout viewportPage title='Article'>
+		<Box sx={{ 
+			display: "flex",
+			flexDirection: "column",
+			justifyContent: "center",
+			alignItems: "center",
+			height: "calc(100% - 60px)"
+			}}>
+
+
+			{/* box where the article is displayed */}
+
+			<Box  sx={{height: "80%",width: "40%",display:"flex",justifyContent: "center",alignItems: "center",backgroundColor: "#D9D9D9", padding:"20px" }}>
+				<fieldset>
+					<legend >Article </legend>
+					<div style={{ padding: '5px' }}>
+						Article is display here
+						<p><a href="https://www.example.com" target="_blank" rel="noopener noreferrer">Visit Website</a></p>
+					</div>
```

Comment:
Please adapt in a way, that we are able to pass the article dynamically into the page.
maybe you can use an demo object in the function that we can replace later with a correct article

Reply:
I have fixed that, check if that's what you want